docs/experiments-disk12.md

```diff
@@ -5,7 +5,8 @@ See http://trec.nist.gov/data/test_coll.html
 **Indexing**:
 
 ```
-nohup sh target/appassembler/bin/IndexCollection -collection Trec -input /path/to/disk12/ \
+nohup sh target/appassembler/bin/IndexCollection -collection TrecCollection \
+ -generator JsoupGenerator  -input /path/to/disk12/ \
  -index lucene-index.disk12.pos -threads 32 -positions -optimize \
  2> log.disk12.pos.emptyDocids.txt 1> log.disk12.pos.recordCounts.txt &
```

Comment:
just to be consistent, simplify the log re-direct as in the CW12 documentation?
same below